innovation.scss

```diff
@@ -1231,6 +1231,14 @@
             bottom: 7px;
             right: 7px;
         }
+        &.top_right_icon {
+            top: 7px;
+            right: 7px;
+        }
+        &.top_center_icon {
+            top: 7px;
+            left: 72px;
+        }
```

Comment:
Probably should plop this onto the .L section too.

Reply:
Done.